playground/api/v1/api.proto

```diff
@@ -31,11 +31,15 @@ enum Sdk {
 
 enum Status {
   STATUS_UNSPECIFIED = 0;
-  STATUS_EXECUTING = 1;
-  STATUS_FINISHED = 2;
-  STATUS_ERROR = 3;
+  STATUS_VALIDATING = 1;
```

Comment:
PLease use buf task in Gradle to generate frontend PB files

Reply:
Added.